### PR TITLE
Reader: Try out showing 3 tags max on stream cards

### PR DESCRIPTION
**Pull request by blowery**
Also support an arbitrarily long set of tags and prevent the avatar from collapsing

### CODE REVIEW

**Comment by blowery** [comment]
before
following_ _reader_ _wordpress_com

after
following_ _reader_ _wordpress_com

**Comment by fraying** [comment]
If we're gonna do this, it should match the formatting on fullpost:

screen shot 2017-03-10 at 1 11 06 pm

**Comment by fraying** [comment]
I'm concerned about adding this much visual noise. Can we try this with up to 3 tags?

**Comment by fraying** [comment]
Looks like the end fade isn't happening soon enough at some narrow widths.

screen shot 2017-03-10 at 1 35 37 pm

**Comment by blowery** [comment]
@fraying could you give this another poke?

**Comment by fraying** [comment]
Would it be possible to release this as a 50/50 AB test? I'm curious to see what difference the three tags would make.

**Comment by bluefuton** [comment]
I like the branch name :D

Adding an A/B test now.

**Comment by fraying** [comment]
Thanks, @bluefuton!

**Comment by blowery** [comment]
just validating the abtest took. will ship after.

thanks @bluefuton for adding the abtest!

File: client/blocks/reader-post-card/byline.jsx [hunk 1]
```diff
@@ -71,8 +71,9 @@ class PostByline extends React.Component {
 		const streamUrl = getStreamUrl( feedId, siteId );
 		const siteIcon = get( site, 'icon.img' );
 		const feedIcon = get( feed, 'image' );
-		const tagsInOccurrenceOrder = take( values( post.tags ), TAGS_TO_SHOW );
```
Comment (samouri) [inline client/blocks/reader-post-card/byline.jsx]: nice spot, i didn't notice this bug.

**Comment by samouri** [comment]
changes LGTM.  testing now

**Pull request merged**